ui-particles-angular/projects/ui-policy-studio-angular/src/lib/gio-policy-studio.apim.stories.ts

```diff
@@ -37,9 +39,21 @@ export default {
     [endpointsInfo]="endpointsInfo"
     [commonFlows]="commonFlows"
     [plans]="plans"
+    (commonFlowsChange)="commonFlowsChange($event)"
+    (plansToUpdate)="plansToUpdate($event)"
     >
     </gio-policy-studio></div>`,
-    props,
+    props: {
+      ...props,
+      commonFlowsChange: (event: Flow[]) => {
+        console.info('commonFlowsChange', event);
```

Comment:
Is console necessary if you are also creating an action?

Reply:
Yes sometime i use it to debug 🙈